data_steward/retraction/retract_utils.py

```diff
@@ -159,6 +159,9 @@ def is_labeled_deid(client, project_id, dataset_id):
     :param dataset_id: Identifies the dataset
     :return: Boolean indicating if the dataset is labeled a deid dataset or None if unlabeled
     """
+    if not client:
+        return None
+
```

Comment:
This function states that `None` indicates the dataset to be unlabeled, but an undefined client can lead to the same response. Should this raise an error instead?

Reply:
I see, this makes sense. I think it would be helpful to add this as a comment saying this returns a None for no-client to trigger a name check.

Reply:
👍 Yes, I will be happy to add that.